modules/ROOT/pages/sleep-processor.adoc

```diff
@@ -0,0 +1,32 @@
+= Sleep Processor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Sleep processor allows you to pause your test for a specific time. You can use this processor to simulate a delay in the processing of the flow:
+
+[source,xml,linenums]
+----
+<munit-tools:sleep time="100" timeUnit="MILLISECONDS"/>
+----
+
```

Comment:
Do we really need a table for just two items that are terms + definitions?
Suggest using definition list; see style ref.